proxy/update_proxy.sh

```diff
@@ -66,13 +73,7 @@ sed -e "s/@VERSION@/$VERSION/g" \
 
 sed -e "s/@VERSION@/$VERSION/g" debian.control-template > debian.control
 
-if [ -z "$ORIGINAL_VERSION" ]; then
-    sed "s/@VERSION@/$VERSION/g;" _service-template > _service
-else
-    sed "s/@VERSION@/$ORIGINAL_VERSION/g;" _service-template > _service
-fi
-
-[ -n "$1" ] && sed -e "s/@PARENT_TAG@/$VERSION/" -i _service || :
+[ -n "$1" ] && sed "s/@REVISION@/$OBS_REVISION/" _service-template > _service || sed "s/@REVISION@/$VERSION/"  _service-template > _service
```

Comment:
Add an explanation here in a comment, why we are using two variables to replace the same value.

Reply:
@gorozco1 Done